src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/WebUI.cs

```diff
@@ -89,6 +118,9 @@ public void Authenticate(Uri authorizationUri, Uri redirectUri, CallState callSt
         private void CallbackMethod(AuthorizationResult result)
         {
             SetAuthorizationResult(result);
+            inBackground = false;
+            DidEnterBackgroundNotification.Dispose();
```

Comment:
will dispose also unregister from NotificationCenter?

Reply:
pls. see also OnMoveToForeground... shouldn't that also clean the background task and clear the notifications?

---
In reply to: [188468356](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1053#discussion_r188468356) [](ancestors = 188468356)